designs/administrator/4.3/quay-image-security/index.md

```diff
@@ -0,0 +1,19 @@
+---
+parent: Administrator
+version: 4.3
+---
+
+# Quay Image Security Integration
+
+Users can now install the Container Security Operator to bring Quay and Clair metadata into OpenShift. Installing this operator enables cluster administrators to monitor known container image vulnerabilities in pods running on their Kubernetes cluster. After deployment, there is no additional configuration for the operator to start querying the container registry for vulnerability data. If the registry supports image scanning, like Quay, then the vulnerabilities found will be exposed to the user by populating the **ImageManifestVuln resource** list and surfacing it on the cluster dashboard.
+
+## Vulnerability details on the cluster dashboard
+
+![Cluster dashboard security card 1](img/image1-73.png)
+- Once the Container Security Operator has been installed, the **Quay Image Security** status is shown on the cluster dashboard Status card.
+- The card will list the number of vulnerabilities found and will provide a link to access more detailed information on the type of vulnerabilities.
+- Clicking on the link opens a popover with the breakdown of vulnerability by severity. Critical and high vulnerabilities will be listed first, followed by medium and lower risk. Fixable vulnerabilities are listed with two links.
+- One link is the vulnerability name that goes out to the vulnerability view for the Quay instance the image is hosted on (for example, Quay.io), and the other is a link to view the Image Manifest Vulnerability resource details in the affected namespace.
+
+![Cluster dashboard security card 5](img/image3-55.png)
```

Comment:
@itsptk couple questions on this image:
- is the icon associated with the severity?  If so, should the icon be placed before the severity label ?
- The second column (Fixable) looks disabled, is that the case?
- Would it be useful for each of the # of fixable vulnerabilities to have it's own link?  For example if there were 12 High, it would filter for the High that are fixable?

Reply:
> * is the icon associated with the severity?  If so, should the icon be placed before the severity label ?

I agree that something might be up with the order here. We were considering moving the severity icon to the left, depending on how many changes we can make in 4.4 bug fixes. Otherwise may rework this in 4.5.

> * The second column (Fixable) looks disabled, is that the case?

It wouldn't be disabled, it's using a lighter styling that some of the other status popovers are using. Another example can be seen here: https://docs.google.com/document/d/1YfS8QXLvm1Iw2M-9P3dPNrmS6rOHJkMbqAbbLYO53QE/edit#bookmark=id.7oxj544do4uf

> * Would it be useful for each of the # of fixable vulnerabilities to have it's own link?  For example if there were 12 High, it would filter for the High that are fixable?

This seems like a decent idea to me, and I will mention when reviewing this design. That would also make the styling of these a link, which could help with the comment above.

Thanks a lot for the feedback!